### PR TITLE
docs: add connect and query examples for java/c/python

### PR DESCRIPTION
**Pull request by bluestreak01**

### CODE REVIEW

File: docs/develop/query-data.md [hunk 1]
```diff
@@ -216,5 +219,109 @@ func checkErr(err error) {
...
+import psycopg2
+
+try:
+   connection = psycopg2.connect(user="admin",
+                                  password="quest",
+                                  host="127.0.0.1",
+                                  port="8812",
+                                  database="postgres_db")
+   cursor = connection.cursor()
+   postgreSQL_select_Query = "select * from test"
```
Comment (mpsq) [inline docs/develop/query-data.md:303]: can we use the query `SELECT x FROM long_sequence(5);` and show its result? This is the one I used for other languages

File: docs/develop/query-data.md [hunk 2]
```diff
@@ -216,5 +219,109 @@ func checkErr(err error) {
...
+    exit(1);
+}
+int main() {
+    PGconn *conn = PQconnectdb("host=localhost user=admin password=quest port=8812 dbname=testdb");
+    if (PQstatus(conn) == CONNECTION_BAD) {
+        fprintf(stderr, "Connection to database failed: %s\n",
+            PQerrorMessage(conn));
+        do_exit(conn);
+    }
+    PGresult *res = PQexec(conn, "select 1");
```
Comment (mpsq) [inline docs/develop/query-data.md:242]: can we use the query `SELECT x FROM long_sequence(5);` and show its result? This is the one I used for other languages

**Review by mpsq — CHANGES_REQUESTED** [review]
Also, the C code does not seem to be formatted, maybe we can use `clang-format`?

File: docs/develop/query-data.md (hunk 1 above)
Comment (bluestreak01) [reply docs/develop/query-data.md:303]: that is a good idea, i was conscious of table not being there

File: docs/develop/query-data.md [hunk 3]
```diff
@@ -216,5 +219,108 @@ func checkErr(err error) {
...
+        do_exit(conn);
+    }       
+    int rows = PQntuples(res);
+    for(int i=0; i<rows; i++) {
+        printf("%s\n", PQgetvalue(res, i, 0));
+    }        
+    PQclear(res);
+    PQfinish(conn);
+    return 0;
+}
```
Comment (mpsq) [inline docs/develop/query-data.md:255]:
```suggestion
// compile with
// g++ libpq_example.c -o libpq_example.exe  -I pgsql\include -L dev\pgsql\lib
// -std=c++17  -lpthread -lpq
#include <libpq-fe.h>
#include <stdio.h>
#include <stdlib.h>
void do_exit(PGconn *conn) {
  PQfinish(conn);
  exit(1);
}
int main() {
  PGconn *conn = PQconnectdb(
      "host=localhost user=admin password=quest port=8812 dbname=testdb");
  if (PQstatus(conn) == CONNECTION_BAD) {
    fprintf(stderr, "Connection to database failed: %s\n",
            PQerrorMessage(conn));
    do_exit(conn);
  }
  PGresult *res = PQexec(conn, "SELECT x FROM long_sequence(5);");
  if (PQresultStatus(res) != PGRES_TUPLES_OK) {
    printf("No data retrieved\n");
    PQclear(res);
    do_exit(conn);
  }
  int rows = PQntuples(res);
  for (int i = 0; i < rows; i++) {
    printf("%s\n", PQgetvalue(res, i, 0));
  }
  PQclear(res);
  PQfinish(conn);
  return 0;
}
```

File: docs/develop/query-data.md [hunk 4]
```diff
@@ -216,5 +219,108 @@ func checkErr(err error) {
...
+
+except (Exception, psycopg2.Error) as error :
+    print ("Error while fetching data from PostgreSQL", error)
+
+finally:
+    #closing database connection.
+    if(connection):
+        cursor.close()
+        connection.close()
+        print("PostgreSQL connection is closed")
```
Comment (mpsq) [inline docs/develop/query-data.md:321]:
```suggestion
import psycopg2
try:
    connection = psycopg2.connect(user="admin",
                                  password="quest",
                                  host="127.0.0.1",
                                  port="8812",
                                  database="postgres_db")
    cursor = connection.cursor()
    postgreSQL_select_Query = "SELECT x FROM long_sequence(5);"
    cursor.execute(postgreSQL_select_Query)
    print("Selecting rows from test table using cursor.fetchall")
    mobile_records = cursor.fetchall()

    print("Print each row and it's columns values")
    for row in mobile_records:
        print("y = ", row[0], "\n")
except (Exception, psycopg2.Error) as error:
    print("Error while fetching data from PostgreSQL", error)
finally:
    #closing database connection.
    if (connection):
        cursor.close()
        connection.close()
        print("PostgreSQL connection is closed")
```

File: docs/develop/connect.md [hunk 5]
```diff
@@ -101,6 +104,91 @@ docker run -it --rm --network=host -e PGPASSWORD=quest postgres psql -h localhos
...
+    print("You are connected to - ", record,"\n")
+
+except (Exception, psycopg2.Error) as error :
+    print ("Error while connecting to PostgreSQL", error)
+finally:
+    #closing database connection.
+        if(connection):
+            cursor.close()
+            connection.close()
+            print("PostgreSQL connection is closed")
```
Comment (mpsq) [inline docs/develop/connect.md:189]:
```suggestion
import psycopg2
try:
    connection = psycopg2.connect(user="admin",
                                  password="quest",
                                  host="127.0.0.1",
                                  port="8812",
                                  database="postgres_db")

    cursor = connection.cursor()
    # Print PostgreSQL Connection properties
    print(connection.get_dsn_parameters(), "\n")

    # Print PostgreSQL version
    cursor.execute("SELECT version();")
    record = cursor.fetchone()
    print("You are connected to - ", record, "\n")

except (Exception, psycopg2.Error) as error:
    print("Error while connecting to PostgreSQL", error)
finally:
    #closing database connection.
    if (connection):
        cursor.close()
        connection.close()
        print("PostgreSQL connection is closed")
```

**Review by mpsq — CHANGES_REQUESTED** [review]

**Review by mpsq — APPROVED** [review]

**Pull request merged**